snippets/text/testnet/connect.md

```diff
@@ -43,3 +43,15 @@ wss://wss.testnet.moonbeam.network
 ### Chain ID
 
 For the Moonbase Alpha TestNet the chain ID is: `1287`.
+
+### Relay Chain
+
+To connect to the Moonbase Alpha relay chain, managed by PureStake, you can use the following DNS(s):
+
+```
+https://wss-relay.testnet.moonbeam.network
```

Comment:
are you sure about this HTTP endpoint? I think there is only a WSS endpoint available (hence the wss-relay...)